Doc/library/io.rst

```diff
@@ -728,6 +728,15 @@ than raw I/O does.
 
       Return :class:`bytes` containing the entire contents of the buffer.
 
+   .. method:: peek(size=1, /)
+
+      Return bytes from the current position onwards but without advancing the
+      position. If the size argument is less than one or larger than the number
+      of available bytes, a copy of the buffer from the current position until
+      the end is returned.
```

Comment:
Reflow according to SemBr.

```suggestion
      Return bytes from the current position onwards without advancing the position.
      If the size argument is less than one,
      or larger than the number of available bytes,
      a copy of the buffer from the current position until the end is returned.
```

Reply:
Good suggestion. I like SemBr, but the rest of the file did not use it.

I applied the change without the added comma, which is not needed as far as I know.